src/hotspot/share/gc/g1/g1Policy.cpp

```diff
@@ -445,10 +445,7 @@ void G1Policy::record_full_collection_end() {
   // Consider this like a collection pause for the purposes of allocation
   // since last pause.
   double end_sec = os::elapsedTime();
-  double full_gc_time_sec = end_sec - _full_collection_start_sec;
-  double full_gc_time_ms = full_gc_time_sec * 1000.0;
-
-  _analytics->update_recent_gc_times(end_sec, full_gc_time_ms);
+  update_pause_time_stats(_full_collection_start_sec, end_sec);
```

Comment:
This will now call `_analytics->compute_pause_time_ratios(end_time_sec, pause_time_ms);` for full collections. Is this intentional? I think this should be good, but this is another (bigger) change in behavior.